deployment/src/main/terraform/modules/data-query/variables.tf

```
@@ -1,11 +1,16 @@
-variable data_query_name {}
+variable ais_raw_catalog_database_name {}
```

Rename to be general as it isn't just raw data in the database

```suggestion
variable ais_catalog_database_name {}
```